gensrc/proto/internal_service.proto

```diff
@@ -89,6 +89,14 @@ message PTabletWriterOpenResult {
     required PStatus status = 1;
 };
 
+message DeltaWriterOpenRequest{
```

Comment:
better rename it to "PartitionOpenRequest" since we do not want to expose DeltaWriter details to upstream BE